codalib/xsdatetime.py

```diff
@@ -0,0 +1,189 @@
+from datetime import datetime, tzinfo, timedelta
+
+from pytz import timezone, utc as utc_tz
+
+# Constants for time parsing/formatting
+# This is a stub
+XSDT_FMT = "%Y-%m-%dT%H:%M:%S"
+# This should never change
+XSDT_TZ_OFFSET = 19
+# The str value to use for default local timezone
+# In localization operations
+DEFAULT_LOCAL_TZ = "US/Central"
```

Comment:
I think it would be good. I'm already using `pytz`. I opted for this route because a) it's a string that can be changed by library users with `set_default_local_tz` and b) the library is presently only used locally.

It might make more sense to go the `tzlocal` route, because the basic behavior would be to "just work" for a library end-user. The only downside might be that a programmer might not know with certainty what the local timezone is set to on production machines. But I suspect that's not a serious worry.